routes/index.js

```diff
@@ -37,6 +37,16 @@ if (enableLegacyProxy) {
           endpoint: '/resources/extsimilar',
           methods: ['get'],
           authed: true
+        },
+        {
+          endpoint: '/values/*',
+          methods: ['get', 'post'],
+          authed: true
+        },
+        {
+          endpoint: '/config/query/all', // NOTE: allows get on all extensions
```

Comment:
Allows getting the 'all' search options..

Reply:
I think it would be nice if we would expose proper rest endpoints for both of these. so, maybe `/search/all/config` for the latter, and `/search/all/values` for the first. It might be possible to leverage this work to rewrite those calls to other urls against ML: https://github.com/marklogic-community/grove-node/pull/29

From top of my head, something like:

    route.use('/search/' + type, routeFactory.defaultExtensionRoute({
        authProvider,
        actions: {
            config: {
                uri: '/v1/config/query/all',
                GET: function(body, params, req){
                    return {
                        method: req.method,
                        body: '',
                        params: {}
                    }
                }
            }
        }

It may be necessary to patch either the default search or default ext route to fall through, otherwise one or the other might not get reached..

Reply:
Could you have a play with that @janmichaelyu ?